demo.html

```diff
@@ -51,7 +53,8 @@
                         document.body.appendChild(imgObject);
                     });
             };
-            img.src = "https://placekitten.com/600/400";
+            
+            img.src = "https://upload.wikimedia.org/wikipedia/en/7/7d/Lenna_%28test_image%29.png";
```

Comment:
```suggestion
            img.src = "https://placecats.com/louie/600/400";
```
Thanks a lot for raising the issue! Looks like https://placekitten.com has been down for months already. I'd like to stick to the cat theme, though.

Reply:
Sure enough 😉 Just used Lena as an example as it is common for some image processing demos 😄